azurerm/internal/services/kusto/resource_arm_kusto_cluster.go

```diff
@@ -122,13 +132,18 @@ func resourceArmKustoClusterCreateUpdate(d *schema.ResourceData, meta interface{
 	}
 
 	location := azure.NormalizeLocation(d.Get("location").(string))
+	enableDiskEncryption := d.Get("enable_disk_encryption").(bool)
+	enableStreamingIngest := d.Get("enable_streaming_ingest").(bool)
 
 	sku, err := expandKustoClusterSku(d)
 	if err != nil {
 		return err
 	}
 
-	clusterProperties := kusto.ClusterProperties{}
+	clusterProperties := kusto.ClusterProperties{
+		EnableDiskEncryption:  utils.Bool(enableDiskEncryption),
```

Comment:
We could do this here
```suggestion
		EnableDiskEncryption:  utils.Bool(d.Get("enable_disk_encryption").(bool)),
```